docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/eksctl.md

```diff
@@ -0,0 +1,288 @@
+
+1. Figure out the VPC that is associated with the cluster
+   - Either via the [AWS UI](https://console.aws.amazon.com/eks/home)
+   - Or CLI - `aws ec2 describe-vpcs` and look for the VPC that contains the tag
+   ```yaml
+   {
+       "Key": "alpha.eksctl.io/cluster-name",
+       "Value": "<clusterName>"
+   },
+   ```
+2. Copy the `vpcId` to use for the next step
+3. Create security group within the VPC for the aurora postgres connection
+
+```shell
+aws ec2 create-security-group \
+  --group-name aurora-postgres-sg \
+  --description "Security Group to allow the k8s cluster to connect to aurora postgres" \
+  --vpc-id <vpcId>
+```
+
+4. Copy the resulting output of the `groupId`
+5. Create security ingress rule to allow access to postgres
+
+```shell
+aws ec2 authorize-security-group-ingress \
+  --group-id <groupId \
+  --protocol tcp \
+  --port 5432 \
+  --cidr 10.192.0.0/16
+```
+
+5. Create db subnet group to associate RDS with within the existing VPC
+
+```shell
+aws rds create-db-subnet-group \
+    --db-subnet-group-name camunda-postgres \
+    --db-subnet-group-description "Subnet for Camunda Postgres" \
+    --subnet-ids subnet-0659823f3a8fa383c subnet-0f435ed77e62a13be subnet-06de5f9aa67849b1c
+```
+
+6. Create RDS cluster within private subnet of the VPC. More configuration options can be found in the [AWS documentation](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/rds/create-db-cluster.html).
+
+```shell
+aws rds create-db-cluster \
+    --db-cluster-identifier <dbCluster> \
+    --engine aurora-postgresql \
+    --engine-version 15.2 \
+    --master-username <username> \
+    --master-user-password <password> \
+    --vpc-security-group-ids sg-004e773e9ac8369a9 \
+    --availability-zones <zones> \ # eu-central-1a eu-central-1b
+    --database-name <dbName> \
+    --db-subnet-group-name camunda-postgres
+```
+
+7. Wait for RDS cluster to be ready
+
+```shell
+aws rds wait db-cluster-available \
+    --db-cluster-identifier <dbCluster>
+```
+
+8. Create database instance within the cluster. More configuration options can be found in the [AWS documentation](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/rds/create-db-instance.html).
+
+```shell
+aws rds create-db-instance \
+    --db-instance-identifier <dbInstance> \
+    --db-cluster-identifier <dbCluster> \
+    --engine aurora-postgresql \
+    --engine-version 15.2 \
+    --no-publicly-accessible \
+    --db-instance-class db.t3.medium
+```
+
+9. Wait for instance to be ready
+
+```shell
+aws rds wait db-instance-available \
+    --db-instance-identifier test-instance
+```
+
+### Connection debugging
+
+1. Create Kubernetes pod to check the in-cluster access
+
+```shell
+kubectl run ubuntu --rm -i --tty --image ubuntu -- bash
+```
+
+2. Install required dependencies
+   Install
+
+```shell
+apt update && apt install -y postgresql-client
+```
+
+3. Connect to Postgresql DB
+
+```shell
+psql \
+  --host=<dbHost> \
```

Comment:
I think it should document how to define `dbHost` (which one to use - cluster (RW and/or RW) and/or the instance one?)

Reply:
Added aws cli commands to automatically retrieve the value, obviously of the writer instance.
Also added environment variables to the kubectl run command to forward the values to the created pod for easier consumption.